Sources/Internal/ColorViewRepresentable.swift

```diff
@@ -15,13 +15,17 @@
 
 import UIKit
 
-extension NSDirectionalEdgeInsets: Hashable {
+enum ColorViewRepresentable: CalendarItemViewRepresentable {
 
-  public func hash(into hasher: inout Hasher) {
-    hasher.combine(top)
-    hasher.combine(leading)
-    hasher.combine(bottom)
-    hasher.combine(trailing)
```

Comment:
removed this `Hashable` conformance - we shouldn't be conforming types we don't own with protocols we don't own, since a new iOS version could introduce its own conformance, which would be a compile-time error.